dandi/tests/fixtures.py

```diff
@@ -150,6 +150,46 @@ def simple3_nwb(
     )
 
 
+@pytest.fixture(scope="session")
+def simple4_nwb(
+    simple1_nwb_metadata: Dict[str, Any], tmp_path_factory: pytest.TempPathFactory
+) -> str:
+    """
+    With, subject, subject_id, species, but including data orientation ambiguity,
+    the only currently non-critical issue in the dandi schema for nwbinspector validation:
+    NWBI.check_data_orientation
+    https://github.com/NeurodataWithoutBorders/nwbinspector/blob/
+    54ac2bc7cdcb92802b9251e29f249f155fb1ff52
+    /src/nwbinspector/internal_configs/dandi.inspector_config.yaml#L10
+    """
+    from datetime import datetime, timezone
```

Comment:
Why not put this at the top of the file?

Reply:
it's just the one fixture using it... is it the project-wide convention to always import at the top?

Reply:
* There's already an import from `datetime` at the top of the file, so you aren't accomplishing anything by placing the import here.
* It's good Python policy to always put imports at the top of the file unless you have a good reason (like circular imports) to do otherwise.  The only dandi-specific reason is avoiding unconditional "heavy" imports, but that doesn't apply to `datetime`.

Reply:
ok :) [d0ef688](https://github.com/dandi/dandi-cli/pull/1162/commits/d0ef688dfb07d534740fccf89a2f3a32f449a7f9)